PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+## Description
+
+Please include a summary of the change
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have followed the agreed [Version Control Practices](https://confluence.niis.org/pages/viewpage.action?spaceKey=XRDDEV&title=Version+Control+Practices)
+- [ ] My changes generate no new warnings or errors (e.g. Javascript console, Java stdout)
+- [ ] I have made corresponding changes to the documentation
+- [ ] The new code has sufficient test coverage
+- [ ] The build, unit and integration tests pass
+- [ ] There is a link to a successful Jenkins build
+- [ ] No new npm audit issues, or needed "accepted audit vulnerabilities" have been added
+  - to [tracking table in Confluence](https://confluence.niis.org/display/XRDDEV/Accepted+npm+audit+vulnerabilities)
+  - to [audit-resolve.json](https://confluence.niis.org/display/XRDDEV/Front-end+build+process#Front-endbuildprocess-Resolvingauditfailures) file in version control
+  - you have crosschecked that these match  
+- [ ] If I fixed npm audit issues, I have updated "accepted audit vulnerabilities" as described above
+- [ ] New backlog items that have been created (such as "not implementing this acceptance criteria now") are mentioned + linked in Jira task comments
+- [ ] All task outputs (PR, documentation, UI design, etc) is listed in a Jira comment so that it is easy for reviewer to check
+- [ ] Deviations from acceptance criteria listed in comments (also if "this criteria was removed")
+
```

Comment:
Template contains links to internal Confluence. However this checklist is presented also to external parties that make pull requests.

Reply:
removed PR template

Reply:
will add separate jira ticket for this https://jira.niis.org/browse/XRDDEV-723?focusedCommentId=12716&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-12716